### ISSUE TITLE
Adhere to WordPress Coding Standards

### ISSUE THREAD

**Issue by GaryJones** [WordPress/wordpress-importer#30]
A quick setup of PHPCS + WPCS + PHPCompatibility shows that this plugin has a few code standards violations:

```
PHP CODE SNIFFER REPORT SUMMARY
----------------------------------------------------------------------
FILE                                                  ERRORS  WARNINGS
----------------------------------------------------------------------
phpunit/tests/import.php                              29      3
phpunit/tests/parser.php                              30      0
phpunit/tests/postmeta.php                            21      0
phpunit/tests/base.php                                10      1
phpunit/bootstrap.php                                 10      1
wordpress-importer.php                                3       1
src/wordpress-importer.php                            1361    144
src/parsers.php                                       242     163
----------------------------------------------------------------------
A TOTAL OF 1706 ERRORS AND 313 WARNINGS WERE FOUND IN 8 FILES
----------------------------------------------------------------------
PHPCBF CAN FIX 1478 OF THESE SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

I'm happy to set up a minimal `composer.json` (just `require-dev`, which can be added to later) that pulls in the code standards dependencies, and then work through the violations with the aim to have 0 violations at the end.

**Comment by ocean90** [comment]
I like that idea. I think that would make it's easier to contribute to this project just like for core.

**Comment by jrfnl** [comment]
Fixed in #75
